tutorials/hyperspectral/Work-With-Hyperspectral-Data-In-R.Rmd

```diff
@@ -105,33 +112,32 @@ library(rhdf5)
 library(rgdal)
 
 # set working directory to ensure R can find the file we wish to import and where
-# we want to save our files. Be sure to move the downloaded into your working directory!
-wd="~/Desktop/Hyperspectral_Tutorial/"
+# we want to save our files. Be sure to move the download into your working directory!
+wd="~/Desktop/Hyperspectral_Tutorial/" #This will depend on your local environment
 setwd(wd)
 
+# Define the file name to be opened
+f <- paste0(wd,"NEON_hyperspectral_tutorial_example_subset.h5")
 ```
 
 <div id="ds-dataTip" markdown="1">
 <i class="fa fa-star"></i> **Data Tip:** To update all packages installed in 
 R, use `update.packages()`.
 </div>
 
-```{r view-file-strux }
-
-# Define the file name to be opened
-f <- paste0(wd,"NEONDSImagingSpectrometerData.h5")
+```{r view-file-strux, eval=FALSE, comment=NA}
 # look at the HDF5 file structure 
-h5ls(f,all=T) 
-
+View(h5ls(f,all=T))
 ```
 
-When you look at the structure of the data, take note of the `map info` dataset, 
-the `spatialInfo` group, and the `wavelength` and `Reflectance` datasets. The 
-`spatialInfo` folder contains the spatial attributes of the data including its 
-Coordinate Reference System (CRS). The CRS documents how the data are physically 
-located on the Earth. The `wavelength` dataset contains the middle wavelength 
-values for each band in the data. The reflectance dataset contains the image data 
-that we will use for both data processing and visualization. 
+When you look at the structure of the data, take note of the `map info` dataset,
```

Comment:
hmmm - do we want to use the code denotation to represent the data/components of the HDF5? This is a different use than we (currently) typically use.  Thoughts?  Maybe move it italics?

Reply:
I do get trigger happy with the backticks to denote code. I think this would be best changed to quotes, and will do so in this latest version.